internal/flex/framework.go

```diff
@@ -156,6 +156,21 @@ func BoolToFrameworkLegacy(_ context.Context, v *bool) types.Bool {
 	return types.BoolValue(aws.ToBool(v))
 }
 
+// EnumStringToFramework converts and enum string to a Framework String value.
+// An empty string is converted to a null String.
+func EnumStringToFramework(_ context.Context, v string) types.String {
+	if v == "" {
+		return types.StringNull()
+	}
+	return types.StringValue(v)
+}
+
+// EnumStringToFrameworkLegacy converts and enum string to a Framework String value.
+// An empty string is left as an empty String.
+func EnumStringToFrameworkLegacy(_ context.Context, v string) types.String {
+	return types.StringValue(v)
+}
+
```

Comment:
Looking at the way these are used they should be named `StringValueToFramework(Legacy)`.
Maybe we can make them generic on `~string` so that the `string()` conversion isn't needed when calling?